EventsExpress/ClientApp/src/components/event/event-form.js

```diff
@@ -24,13 +24,7 @@ const imageIsRequired = value => (!value ? "Required" : undefined);
 const { validate } = Module;
 
 class EventForm extends Component {
-    constructor(props) {
-        super(props);
-        this.state = { 
-            imagefile: [],
-            checked: false
-        };
-    }
+    state = { imagefile: [], checked: false, photoCropped: false };
```

Comment:
You already have `photoCropped` state in the `DropZone` component. Please remove this one.

Reply:
The `imageFile` is used only for a specific component to store state. This state is not required, since there will be a state coming from `initialValues` or `value` of a react-redux form. Consider picking an image from there rather than from state.